src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -33,9 +36,9 @@
  * @author Jonathan Fuerth
  * @author Ognjen Bubalo
  */
-public final class JacocoBuildAction extends CoverageObject<JacocoBuildAction> implements HealthReportingAction, StaplerProxy, Serializable {
+public final class JacocoBuildAction extends CoverageObject<JacocoBuildAction> implements HealthReportingAction, StaplerProxy, Serializable, RunAction2 {
 
-	public final AbstractBuild<?,?> owner;
+	public transient Run<?,?> owner;
```

Comment:
this is a backwards incompatable change.  Not that I am trying to suggest a BCT transform